content/authors/michael-mul/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+# View this page at https://digital.gov/authors/michael-mul
+# Learn how to edit our pages at https://workflow.digital.gov
+
+# slug — the specific user-id for an author.
+slug: michael-mul
```

Comment:
Should the slug still include an 'e' at the end?

Reply:
Yes